lambda/repository/lambda_functions.py

```diff
@@ -68,18 +68,18 @@ def _get_cert_path() -> str | bool:
     return rest_api_cert_path
 
 
-def _get_embeddings(provider: str, model_name: str, id_token: str) -> LisaEmbeddings:
+def _get_embeddings(model_name: str, id_token: str) -> LisaOpenAIEmbeddings:
     global lisa_api_endpoint
 
     if not lisa_api_endpoint:
         lisa_api_param_response = ssm_client.get_parameter(Name=os.environ["LISA_API_URL_PS_NAME"])
         lisa_api_endpoint = lisa_api_param_response["Parameter"]["Value"]
 
-    lisa = Lisa(
-        url=lisa_api_endpoint, verify=_get_cert_path(), timeout=60, headers={"Authorization": f"Bearer {id_token}"}
-    )
+    headers = {"Authorization": f"Bearer {id_token}"}
 
-    embedding = LisaEmbeddings(provider=provider, model_name=model_name, client=lisa)
+    embedding = LisaOpenAIEmbeddings(
+        lisa_openai_api_base=lisa_api_endpoint + "/v2/serve", model=model_name, headers=headers
```

Comment:
should we get the apiVersion here from the environment variables (do we have a way to get that, either in the env or in an SSM Param)?

Reply:
I pushed a change to add an environment variable for this. Resolving thread.